apps/condo/domains/acquiring/schema/CalculateFeeForReceiptService.test.js

```diff
@@ -46,6 +47,27 @@ describe('CalculateFeeForReceiptService', () => {
             expect(result.explicitServiceCharge).toBe('0')
         })
     })
+    describe('Check receipt', () => {
+        test('Receipt must exist', async () => {
```

Comment:
It would be better to make the test names more understandable.
For example, 'should throw an error if there is no receipt'.